### PR TITLE
added coverage.ini

### PR DESCRIPTION
**Pull request by javierggt**
## Description

This PR adds a coverage.ini file that is required to run testr with coverage.

## Testing

- [x] Functional testing as described in sot/testr/pull/43

### CODE REVIEW

**Comment by taldcroft** [comment]
I expect this will reveal large gaps in coverage, but I guess there is no point keeping ones head in the sand... 😸

File: coverage.ini [hunk 1]
```diff
@@ -0,0 +1,44 @@
+[run]
+branch = True
+source_pkgs =
+    acis_thermal_check,
+    acispy,
+    backstop_history,
+    prompt_toolkit,
```
Comment (taldcroft) [inline coverage.ini:7]: Remove

File: coverage.ini [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    Chandra,
+    Quaternion,
+    Ska,
```
Comment (taldcroft) [inline coverage.ini:10]: Is this getting all the Ska.* packages?  (And likewise for `Chandra`). Seems a little odd, might be better to call them out explicitly, but I don't feel strongly if this is working.

**Review by taldcroft — APPROVED** [review]
Looks good, let's do it!

File: coverage.ini (hunk 2 above)
Comment (javierggt) [reply coverage.ini:10]: Yes, this is actually a directory, I think. I would leave it.

Comment (taldcroft) [reply coverage.ini:9]: OK

**Review by taldcroft — APPROVED** [review]

**Pull request merged**